templates/GenericTemplate.hs

```diff
@@ -150,8 +150,12 @@ indexShortOffAddr (HappyA# arr) off =
 indexShortOffAddr arr off = arr Happy_Data_Array.! off
 #endif
 
+{-# INLINE unbox_int #-}
 unbox_int IBOX(x) = x
```

Comment:
These functions share a namespace with the user's code, so let's use names prefixed with `happy` to avoid clashes. (I know we don't do this consistently, but we should)